test/wasmBenchmarker/benchmark.py

```diff
@@ -89,8 +90,12 @@ def check_programs(engines, verbose):
     if (verbose): print("Checks done")
 
 
-def get_emcc(verbose):
-    emcc_path = None 
+def get_emcc():
+    emcc_path = shutil.which("emcc")
```

Comment:
Is it a recursive search? If it is it can be very costly if the path contains a lot of directories.